hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -5097,6 +5122,14 @@ private String reconfigureAllowListAllVolumes(String newVal) {
     return String.valueOf(allowListAllVolumes);
   }
 
+  private String reconfOzoneDirDeletingServiceInterval(String newVal) {
```

Comment:
can this validation be registered by the service using this configuration, ie directorydeleteingService, and below logic also do validation, can that be used instead of using pattern if not null., configuration.getTimeDuration(
          OZONE_BLOCK_DELETING_SERVICE_INTERVAL,
          TimeUnit.MILLISECONDS);